bucoffea/vbfhinv/definitions.py

```diff
@@ -273,24 +273,31 @@ def vbfhinv_accumulator(cfg):
     return  processor.dict_accumulator(items)
 
 def vbfhinv_regions(cfg):
+    # 'inclusive'    # 'veto_b',
+    def clean_lists(orig_list, to_remove):
+        return list(filter(lambda x: x not in to_remove, orig_list))
```

Comment:
Thanks for factoring this out to a function. Do you think if we can use a slightly more descriptive name for this though, maybe like`remove_cuts_from_list`? Also feel free to remove the comments on L276 :)